numpy/typing/mypy_plugin.py

```diff
@@ -1,4 +1,34 @@
-"""A module containing `numpy`-specific plugins for mypy."""
+"""A mypy_ plugin for managing a number of platform-specific annotations.
+Its functionality can be split into three distinct parts:
+
+* Assigning the (platform-dependent) precisions of certain `~numpy.number`
+  subclasses, including the likes of `~numpy.int_`, `~numpy.intp` and
+  `~numpy.longlong`. See the documentation on
+  :ref:`scalar types <arrays.scalars.built-in>` for a comprehensive overview
+  of the affected classes. Without the plugin the precision of all relevant
+  classes will be inferred as `~typing.Any`.
+* Removing all extended-precision `~numpy.number` subclasses that are
+  unavailable for the platform in question. Most notably this includes the
+  likes of `~numpy.float128` and `~numpy.complex256`. Without the plugin *all*
+  extended-precision types will, as far as mypy is concerned, be available
+  to all platforms.
+* .. versionadded:: 1.22
```

Comment:
It looks a bit weird, but does seem to render correctly.

Reply:
It does. Why not after the explanation?

Reply:
Mostly because I hadn't considered it.
Done as of https://github.com/numpy/numpy/pull/19295/commits/7cf1b178485d38292f367f8e0c017a64b7ab67b6.